config/initializers/assets.rb

```diff
@@ -11,7 +11,10 @@
 
 Rails.application.config.assets.paths << Rails.root.join("app/assets/xml")
 
+Rails.application.config.assets.paths << Rails.root.join("vendor/assets/javascripts")
+
```

Comment:
These aren't needed, `vendor/assets/javascripts` is already part of the paths where Rails looks for:

From `main`:

```ruby
> Rails.application.config.assets.paths.include?(Rails.root.join("vendor/assets/javascripts").to_s)
=> true
```

```suggestion

```